www/src/pages/de/faq.md

```diff
@@ -0,0 +1,59 @@
+---
+title: FAQ
+description: Häufig gestellte Fragen zu Create T3 App
+layout: ../../layouts/docs.astro
+lang: de
+---
+
+Hier sind einige häufig gestellte Fragen zu `create-t3-app`.
+
+## Was kommt als nächstes? Wie erstelle ich eine App mit diesem Tool?
```

Comment:
Maybe "Was jetzt?" or something like that

Reply:
I prefer the current text. Open for suggestions